Assets/Scripts/Object/AbstractObject.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using Utils;
+
+namespace Object {
+    public abstract class AbstractObject : MonoBehaviour {
+        public float minMoveSpeed;
+        public float maxMoveSpeed;
+        public float minRotSpeed;
+        public float maxRotSpeed;
+
+        private float moveSpeed;
+        private Vector3 targetPosition;
+        private float rotSpeed;
+        private Vector3 rotDirection;
+
+        public void Start() {
+            moveSpeed = Random.Range(minMoveSpeed, maxMoveSpeed);
+            targetPosition = Vector3.zero;
+
+            rotSpeed = maxRotSpeed != 0
+                ? Random.Range(minRotSpeed, maxRotSpeed)
+                : 42F;
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg) 
Is this hard coded number? Maybe it should be defined as constant variable

Reply:
yeah youre right